pmlc/dialect/pxa/tests/autotile.mlir

```diff
@@ -12,5 +11,16 @@ func @dot(%arg0: memref<100x100xf32>, %arg1: memref<100x100xf32>) -> memref<100x
   }
   return %out : memref<100x100xf32>
 }
-// CHECK: affine.parallel
-// CHECK: affine.parallel
+
+// CHECK-LABEL: func @dot
+// CHECK-SAME: (%arg0: memref<100x100xf32>, %arg1: memref<100x100xf32>) -> memref<100x100xf32>
+// CHECK: %1 = affine.parallel (%[[arg2:.*]], %[[arg3:.*]], %[[arg4:.*]]) = (0, 0, 0) to (100, 100, 100) step (10, 10, 10) {
```

Comment:
We should capture the results so that `%1` isn't used directly.

Reply:
Another possibility is that if you don't need `%1` from anything, it's fine to just not check for it.